tests/frontend/screens.test.js

```diff
@@ -24,6 +24,7 @@ describe('Render all screens', () => {
     const {container} = renderWithRedux(<StyledApp/>, {
       history,
       store: createStore(() => ({
+        calendar: {datesLoading:true},
```

Comment:
add a case where there is actuall data here and loading is false.
then coverage will bump to the sky :)